images/cso/Dockerfile

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM --platform=${BUILDPLATFORM} docker.io/alpine/helm:3.15.1 as helm
+FROM --platform=${BUILDPLATFORM} docker.io/alpine/helm:3.15.2 as helm
```

Comment:
don't want helm to be updated this way. 
Ref: https://github.com/SovereignCloudStack/cluster-stack-operator/pull/228